Matter Server: bump to 1.2.6, add time_sync and default_fabric_label options (9.1.0) (#4727)
home-assistant/addons: matter_server/DOCS.md

| log_level            | Logging level of the Matter Server component.                                                           |
| beta                 | Install the latest `matter-server` from npm on startup instead of the bundled version. On failure a warning is logged and the bundled version is started. |
| enable_test_net_dcl  | Enable test-net DCL for PAA root certificates, OTA updates and other device information.                |
| time_sync            | Whether the Matter Server pushes the current time to Matter devices: `auto` (default; on only when the host clock is NTP synchronized), `on` (always; warns if NTP is not synchronized), or `off`. |

> | default_fabric_label | Pin the fabric label (shown on Matter devices, max 32 characters) to this value. Use this if you use the Matter Server from multiple Home Assistant instances, which otherwise each keep pushing their own instance name as the label. When set, changing the label via the WebSocket API is blocked. |

Reviewer: Maybe worth mentioning what the default is?

```suggestion
| default_fabric_label | Pin the fabric label (shown on Matter devices, max 32 characters) to this value. By default the Home Assistant home name is used. This is useful if the Matter Server is used from multiple Home Assistant instances, which otherwise each keep pushing their own home name as the label. When set, changing the label via the WebSocket API is blocked. |
```
